pkgs/top-level/all-packages.nix

```diff
@@ -35935,8 +35935,6 @@ with pkgs;
 
   zotero = callPackage ../applications/office/zotero { };
 
-  zotero_7 = callPackage ../applications/office/zotero/zotero_7.nix { };
```

Comment:
Please also modify `pkgs/top-level/aliases.nix` to point `zotero_7` to `zotero_beta` in order to avoid breakage